application/core/LS_Twig_Extension.php

```diff
@@ -617,11 +617,13 @@ public static function darkencss($cssColor, $grade = 10, $alpha = 1)
      * @param mixed $needle The searched value.
      * @param array $haystack The array.
      * @param bool $strict If the third parameter strict is set to TRUE then the in_array() function will also check the types of the needle in the haystack.
+     * @todo in_array_r is not defined - delete this method?
      */
     function in_multiarray($needle, $haystack, $strict = false)
     {
 
         foreach ($haystack as $item) {
+            /** @psalm-suppress UndefinedFunction */
             if (($strict ? $item === $needle : $item == $needle) || (is_array($item) && in_array_r($needle, $item, $strict))) {
```

Comment:
this is for sure meant to be in_array, would change it. The function also seeems completely unzused right now but some custom survey themes might use it

Reply:
Added to https://bugs.limesurvey.org/view.php?id=18850